docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_customize_build_pipeline.adoc

```diff
@@ -0,0 +1,99 @@
+= Customizing the build pipeline
+
+You can customize the way that {ProductName} builds components, to meet your specific needs, in two ways:
```

Comment:
Nit pick, but we typically lead with reason, then action. So it'd be something like:

To meet your specific needs, customize the way that {ProductName} builds components. You can customize in two ways: